src/store/Reducer.ts

```diff
@@ -1,10 +1,13 @@
 import { Reducer } from "react";
 import { AsyncActionHandlers } from "use-reducer-async";
+import Cookies from "universal-cookie";
 
 import { State } from "./State";
 import * as Actions from "./Actions";
 import { admin } from "./Textile";
 
+export const cookies = new Cookies();
```

Comment:
This is slightly annoying, but seems reasonable. We keep a single "reference" cookies instance here so that we can modify it from both react components, and the store. Otherwise, our provide supplies its own cookies instance, which then doesn't allow us to cause rendering updates from here.